agent_path_prediction/AgentPose[]  goals
---
bool                         success
string                       message
